bootstrap/node.go

```diff
@@ -36,6 +36,7 @@ func NewNode(
 	activeConsensusAlgo consensus.ConsensusAlgoType,
 	benchmarkConsensusRoundRetryIntervalMillisec uint32, // TODO: move all of the config from the ctor, it's a smell
 	transport gossipAdapter.Transport,
+	maxStateHistory uint64,
```

Comment:
are the units here in blocks? if so, maybe improve the name or change from `uint64` to `primitives.blockHeight` to clarify

Reply:
hmm.. since it's in config maybe leave as `uint64` but improve the name.. in the [config](https://github.com/orbs-network/orbs-spec/blob/master/behaviors/config/services.md) in the spec we called it `recent-block-snapshot-num`

Reply:
`stateHistoryRetentionInBlockHeights`

Reply:
done